<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddGraphics</class>
 <widget class="QDialog" name="DialogAddGraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>365</width>
        <height>233</height>
       </rect>
      </property>
      <attribute name="label">
       <string>添加矩形</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>x:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xValue"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yValue"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>宽度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="width">
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>高度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="height"/>
       </item>
       <item>
        <widget class="QPushButton" name="addPointButton">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>224</height>
       </rect>
      </property>
      <attribute name="label">
       <string>添加圆</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>x：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="circleX"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="circleY"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>半径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="radius"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
